process-services/latest/support/index.md

```diff
@@ -59,21 +60,21 @@ our Support team [Alfresco Support Portal](https://support.alfresco.com).
 
 **Third party integrations**
 
-|Integration|1.11|Comment|
+|Integration|2.0|Comment|
 |-----------|----|-------|
 |Elasticsearch 7.6|✓| |
```

Comment:
Elasitcsearch 7.13.2 is supported. Kindly remove older versions

Reply:
Done for 2.0